src/main/java/spoon/support/compiler/SpoonPom.java

```diff
@@ -654,4 +688,39 @@ public String getPath() {
 	public File toFile() {
 		return pomFile;
 	}
+
+	/**
+	 * Additional options for maven invocations.
+	 */
+	@Internal
+	public static class MavenOptions {
```

Comment:
What's the reasoning for making this a nested class in `SpoonPom`? Barring the fact that it's used _by_ `SpoonPom`, it doesn't seem intrinsically related.

Reply:
I wanted to keep this an internal implementation detail and the directory SpoonPom resides in didn't feel adequate (it doesn't have anything to do with compilers). Additionally it is quite closely related so I hoped I could get away with it.

If you feel that's too much of a stretch I can extract it with a more verbose name :)

Reply:
> the directory SpoonPom resides in didn't feel adequate

That we can agree on. It would have to go into an internal subpackage, and that'd put it far away from it's currently singular use. I mostly wanted to see if you'd thought it through, which you have, so let's leave it where it is :)